### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by AndrewJDFerguson** [ApacheFriends/xampp-build#5]
When installing a newer version of XAMPP, there is currently no way, that I can tell, to upgrade the currently installed version to the new version. I get that there might be limitations on this, like trying to upgrade from a really old release, or perhaps if XAMPP components are currently in use. 

Perhaps I am missing some part of how this should be done, but in my scenario, I have multiple MySQL databases setup and need to bring in the user logins and site data to the updated configuration.

**Comment by beltran-rubo** [comment]
That is a great feature but it is also difficult to ensure the upgrade process works well. There is not a single solution, it depends on your app. XAMPP can be installed in different folders so I would suggest to install both versions and migrate your application (database and files) from one version to other.

**Comment by AndrewJDFerguson** [comment]
I agree it is a great feature, and am sure it is not easy to implement. There is a lot of configuration for the pieces that comprise XAMPP, and they are not exactly under XAMPP's control, other than the version to include and perhaps some default values. So, no worries at all. 

Regarding installing to different folders, I actually have done something similar, but rename the existing installation folder to something like xampp816, and let the new version end up in the default folder. Then I have brought over the data, and merged my configurations. I tend to keep XAMPP in its default folder, as WordPress (not XAMPPs issue here) likes to include paths in some of their database entries. I know how to change those if needed, but if I install XAMPP into the default folder, then that part does not need to change. 

Anyhow, I appreciate that you at least considered my suggestion.
